site/docusaurus.config.ts

```diff
@@ -5,8 +5,11 @@ import GenAISamplesDocsPlugin from './src/plugins/genai-samples-docs-plugin';
 
 // This runs in Node.js - Don't use client-side code here (browser APIs, JSX...)
 
-const organizationName = 'openvinotoolkit';
-const projectName = 'openvino.genai';
+// GITHUB_REPOSITORY env var is set by GitHub Actions
+const GITHUB_REPOSITORY = process.env.GITHUB_REPOSITORY || 'openvinotoolkit/openvino.genai';
+const [organizationName, projectName] = GITHUB_REPOSITORY.split('/');
+// GITHUB_REPOSITORY is set by GitHub Actions
+const projectName = `${process.env.GITHUB_REPOSITORY?.split('/')[1] || 'openvino.genai'}`
```

Comment:
Duplicated const will cause an error
```suggestion
```
GitHub does not support code suggestion for both deleted and added lines, so I was not able to change it correctly in my previous comment